.github/workflows/core-dependency-update.yml

```
@@ -0,0 +1,60 @@
+        working-directory: ci/scripts
+        run: node git-operations.mjs configure
+
+      # Extract core version and run the complete pipeline
+      - name: Run core dependency update pipeline
+        working-directory: ci/scripts
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          # Install dependencies
+          npm ci
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)